packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -125,6 +128,11 @@ export class SCWSigner implements Signer {
         return this.storage.loadObject(WALLET_CAPABILITIES_STORAGE_KEY);
       case 'wallet_switchEthereumChain':
         return this.handleSwitchChainRequest(request);
+      case 'wallet_sendCalls':
+        // Temporary solution for demo purposes
+        if (isPresigned(request)) return fetchRPCRequest(request, PERMISSIONS_BACKEND_URL)
```

Comment:
can we just check here
const params = request.params as { capabilities?: { presigned?: boolean } };

this seems unnecessary
<img width="488" alt="image" src="https://github.com/user-attachments/assets/1311e16f-aecf-4597-97df-e4367ecc9323">

Reply:
or do we expect this func to cover other method types as well?